omnibus/files/private-chef-ctl-commands/ha.rb

```diff
@@ -94,6 +94,9 @@
   ha_services = Dir.chdir(running_config['runit']['sv_dir']){Dir.glob('**/keepalive_me').map{|f| File.dirname(f)}}.sort
 
   get_all_services.sort.each do |service_name|
+    # opscode-chef-mover service is only used during an upgrade, and does not need to be running all of the time
+    next if service_name.eql? "opscode-chef-mover"
```

Comment:
We may want to use `omnibus-ctl`'s concept of hidden services here.  In that case, this check would become:

```
next if hidden_services.include?("opscode-chef-mover")
```